WickedEngine/wiJobSystem.cpp

```diff
@@ -14,6 +14,9 @@
 
 #ifdef PLATFORM_LINUX
 #include <pthread.h>
+#if __has_include(<pthread_np.h>)
```

Comment:
pthread_np seems FreeBSD exclusive, so wouldn't it make more sense to use `#if __FREEBSD__` here? Since if we're under BSD, that header is needed for compilation, and not just optional.